File: backend/nightbiteApi/src/main/java/ni/edu/uam/nightbiteapi/dto/PlayerResponse.java
package ni.edu.uam.nightbiteapi.dto;

import lombok.Getter;
import lombok.Setter;
import ni.edu.uam.nightbiteapi.enums.Gender;

import java.time.LocalDateTime;

@Setter
@Getter
public class PlayerResponse {

    private Long id;
    private Long userAccountId;
    private String driverName;
    private Gender gender;
    private String helmetColor;
    private String motorcycleType;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;

    public PlayerResponse(
            Long id,
            Long userAccountId,
            String driverName,
            Gender gender,
            String helmetColor,
            String motorcycleType,
            LocalDateTime createdAt,
            LocalDateTime updatedAt
    ) {
        this.id = id;
        this.userAccountId = userAccountId;
        this.driverName = driverName;
        this.gender = gender;
        this.helmetColor = helmetColor;
        this.motorcycleType = motorcycleType;
        this.createdAt = createdAt;
        this.updatedAt = updatedAt;
    }

}

File: mobile/nightbiteApp/app/src/main/java/ni/edu/uam/nightbiteapp/data/remote/dto/PlayerResponse.kt
package ni.edu.uam.nightbiteapp.data.remote.dto

data class PlayerResponse(
    val id: Long,
    val userAccountId: Long,
    val driverName: String,
    val gender: String,
    val helmetColor: String,
    val motorcycleType: String,
    val createdAt: String? = null,
    val updatedAt: String? = null
)